en/sql/query/json_table.rst

```diff
@@ -0,0 +1,145 @@
+- name type PATH json path [on empty] [on error]: Columns of this type are used to extract json_values from the specified json paths. The extracted json data is then coerced to the specified type.
+  If the path does not exist, json value triggers the on empty clause. The on error clause is triggered if the extracted json value is not coercible to the target type.
+
+  - on empty determines the behavior of JSON_TABLE in case the path does not exist. On empty can have one of the following values:
+
+    - NULL ON EMPTY: the column is set to NULL. This is the default behavior.
+    - ERROR ON EMPTY: an error is thrown
+    - DEFAULT json_string ON EMPTY: json_string will be used instead of the missing value. Column type rules also apply to the default value.
+
+  - on error can have one of the following values:
+
+    - NULL ON ERROR: the column is set to NULL. This is the default behavior.
+    - ERROR ON ERROR: an error is thrown.
+    - DEFAULT json_string ON ERROR: json_string will be used instead of the array/object/json scalar that failed coercion to desired column type. Column type rules also apply to the default value. 
+
+- name type EXISTS PATH json path: this returns 1 if any data is present at the json path location, 0 otherwise.
+
+- NESTED [PATH] json path COLUMNS (column list):
+  The json path is relative to the parent's path. The parent's path is either JSON_TABLE's json path or the path of the parent NESTED [PATH] clause.
+
+
+.. code-block:: sql
+
+    SELECT * FROM JSON_TABLE ('{"a":[1,[2,3]]}', '$.a[*]' COLUMNS ( col INT PATH '$')) as jt;
+::
+
+                       col
+    ======================
+                         1 -- first value found at '$.a[*]' is 1 json scalar, which is coercible to 1
+                      NULL -- second value found at '$.a[*]' is [2,3] json array which cannot be coerced to int, triggering NULL ON ERROR default behavior
+
+Overriding the default on_error behavior, results in a different output from previous example: 
+
+.. code-block:: sql
+
+    SELECT * FROM JSON_TABLE ('{"a":[1,[2,3]]}', '$.a[*]' COLUMNS ( col INT PATH '$' DEFAULT '-1' ON ERROR)) as jt;
+::
+
+                       col
+    ======================
+                         1 -- first value found at '$.a[*]' is '1' json scalar, which is coercible to 1
+                        -1 -- second value found at '$.a[*]' is '[2,3]' json array which cannot be coerced to int, triggering ON ERROR
+
+ON EMPTY example:
+
+.. code-block:: sql
+
+    SELECT * FROM JSON_TABLE ('{"a":1}', '$' COLUMNS ( col1 INT PATH '$.a', col2 INT PATH '$.b', col3 INT PATH '$.c' DEFAULT '0' ON EMPTY)) as jt;
+
+::
+
+             col1         col2         col3
+    =======================================
+                1         NULL            0 
+  Column col2 represents the value found at '$.a' in the given json_doc. Since the path does not exist, ON EMPTY is triggered resulting in NULL as a result.
```

Comment:
I think you can remove exaplantion; the query result is self-explanatory and it doesn't look very good.